clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -48,6 +47,7 @@
     private LakeFSClient lfsClient;
     private int listAmount;
     private FileSystem fsForConfig;
+    private String scheme;
```

Comment:
Is this just a part of `uri`?

Reply:
true, reused from the instance we already got. thanks!